compat/vcbuild/vcpkg_install.bat

```diff
@@ -80,6 +80,11 @@ REM ================================================================
 :sub__install_one
 	echo     Installing package %1...
 
+REM vcpkg may not be reliable on slow, intermittent or proxy connections
+REM e.g. https://social.msdn.microsoft.com/Forums/windowsdesktop/en-US/4a8f7be5-5e15-4213-a7bb-ddf424a954e6/winhttpsendrequest-ends-with-12002-errorhttptimeout-after-21-seconds-no-matter-what-timeout?forum=windowssdk
+REM which explains the hidden 21 second timeout
+REM (last post by Dave : Microsoft - Windows Networking team)
```

Comment:
Is there any particular reason why this is not indented the same as the surrounding lines?

Reply:
That'll probably be local tab settings. It was aligned in my editor (Npp++).

What's the easiest/simplest fix? (I'm about to retire for the night, hopefully)

Reply:
> What's the easiest/simplest fix?

The easiest fix is, as always, to leave the cleaning up to me...

Reply:
> > What's the easiest/simplest fix?

> The easiest fix is, as always, to leave the cleaning up to me...

I wasn't sure if it was just a 5 second job while doing the 1 minute merge, or the other way around? (the latter would be undesirable to say the least ;-) 

Sorry if it was the other way. The overall fix, amend, push, PR update time from my end would take me too far past my fall asleep time....

Reply:
> I wasn't sure if it was just a 5 second job while doing the 1 minute merge, or the other way around?

This is what I did:
1. fetched your branch,
2. checked it out,
3. opened the file in an editor,
4. indented the lines manually,
5. re-wrapped at <= 80 columns per line,
6. looked whether it was the tip commit to amend,
7. after satisfying myself that it was, amended it,
8. looked over the amended commit to verify everything looks all right,
9. force-pushed.

I'm sorry, I did not start a stop watch to measure how long it took me. That's not usually my priority, I focus more on getting it right... but yes, I believe that this must have taken a bit longer than five seconds.

Reply:
Thanks for clarifying. I had expected somehow that the github process was quicker.

It's not always obvious what tooling is available in the different places, especially as many of the Github capabilities are so hard to discover (you mentioned the 'r' key in a git list thread), or possibly non-existent (thread hierarchy summaries).

Reply:
> many of the Github capabilities are so hard to discover (you mentioned the 'r' key in a git list thread)

Just press `?` on a GitHub page (unless your keyboard focus is on a text input box). This is not just GitHub, most web pages I know support this way to see which keyboard shortcuts are available.

Reply:
> I had expected somehow that the github process was quicker.

I could imagine that GitHub Desktop would help here, but I am unsure about the options to indent `.bat` files in their editor.